### ISSUE TITLE
rule-nested-empty-line-before always failed with @keyframes

### ISSUE THREAD

**Issue by Tiendq** [stylelint/stylelint#1945]
`"rule-nested-empty-line-before": "always"` failed even there is an empty line before 100%

`@keyframes test {
  0% {
    transform: rotate(0deg);
  }

  100% {
    transform: rotate(360deg);
  }
}`

**Comment by evilebottnawi** [comment]
@Tiendq format you issue as template

**Comment by jeddy3** [comment]
Thanks for creating this issue but we are closing it as issues need to follow our issue template, so that we can clearly understand your particular circumstances.

Please help us to help you by [recreating the issue](https://github.com/stylelint/stylelint/issues/new) using the template.

**Comment by onlurking** [comment]
Today I've run in the same issue, here's the fix for anyone who finds this issue in the future:

```json
"rule-empty-line-before": ["always-multi-line", { "except": ["first-nested"] }],
```

So, this code is valid:
```css
@keyframes roll {
  0% {
    transform: rotate(0);
  }

  50% {
    transform: rotate(180deg);
  }

  100% {
    transform: rotate(360deg);
  }
}
```
